fe/fe-core/src/test/java/org/apache/doris/nereids/trees/expressions/literal/DateTimeLiteralTest.java

```diff
@@ -443,4 +443,36 @@ void testDateTimeV2Scale() {
                 new DateTimeV2Literal(DateTimeV2Type.of(5), "2016-12-31 23:59:59.999999"),
                 new DateTimeV2Literal("2017-01-01 00:00:00.0"));
     }
+
+    @Test
+    void testRoundFloor() {
+        DateTimeV2Literal literal;
+        literal = new DateTimeV2Literal(DateTimeV2Type.of(6), 2000, 2, 2, 2, 2, 2, 222222);
```

Comment:
should we also test for input which have more than six decimal places?

Reply:
no, DateTimeV2Literal has ensure mircosecond could not have more than six scale